Source/stores.cpp

```diff
@@ -500,10 +505,10 @@ void PrintStoreItem(ItemStruct *x, int l, char iclr)
 			sprintf(tempstr, "%s %i Dex", tempstr, x->_iMinDex);
 		strcat(sstr, tempstr);
 	}
-	AddSText(40, l, 0, sstr, iclr, 0);
+	AddSText(40, l++, 0, sstr, iclr, 0);

```

Comment:
Img guessing your sure l++ doesn't belong right after the function

Reply:
ofc I tested, it's not bin exact otherwise